### PR TITLE
Adding a Django development environment

### LINKED ISSUE #3: Setting up a Django development environment
In this project, we will use Django as our software framework.

### PR DESCRIPTION
**Pull request by Shoval-Ovadia**
- Setup Pipenv to install and manage Django and other Python
dependencies.
- Bootstrapped an empty Django application.
- made the application accessible from a web browser.

closes #3

### CODE REVIEW

**Review by maayanhd — COMMENTED** [review]
never mind, it looks good, I got confused with the rest of the bootstrap part (in case someone already read the unedited comment)

**Review by maayanhd — APPROVED** [review]
looks fine to me, only I suggest linking the PR to the issue by using a work like fixe/closes and the the #issue number, you can see a link appeared in one of the first comments in here: 
> > related issue: #4
> 
> You can automatically link the issue to the PR, see [here](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

**Review by nadavspinco — APPROVED** [review]

File: Vagrantfile [hunk 1]
```diff
@@ -19,6 +19,11 @@ Vagrant.configure("2") do |config|
   end
   config.vm.provision "shell", path: "bootstrap.sh", privileged: false
 	
+
```
Comment (AmirMiller91) [inline Vagrantfile:22]: I think the indentation here should be fixed.

**Review by AmirMiller91 — CHANGES_REQUESTED** [review]
I successfully ran the server and accessed the webpage. The only nitpick I have is the indentation in the Vagrantfile.

File: Vagrantfile [hunk 2]
```diff
@@ -19,6 +19,11 @@ Vagrant.configure("2") do |config|
   end
   config.vm.provision "shell", path: "bootstrap.sh", privileged: false
 	
+	
+	config.vm.network(
+"forwarded_port", guest: 8000, host: 8000, host_ip: "127.0.0.1"
+)
```
Comment (AmirMiller91) [inline Vagrantfile:25]: I think the indentation here should be fixed.

Comment (Shoval-Ovadia) [reply Vagrantfile]: I fixed it.

File: Vagrantfile [hunk 3]
```diff
@@ -19,6 +19,11 @@ Vagrant.configure("2") do |config|
   end
   config.vm.provision "shell", path: "bootstrap.sh", privileged: false
 	
+	
+config.vm.network(
+"forwarded_port", guest: 8000, host: 8000, host_ip: "127.0.0.1"
+)
+
```
Comment (Leeadar) [inline Vagrantfile:26]: I think those lines should be with double space, that way it will be in the do - end block.

Comment (Shoval-Ovadia) [reply Vagrantfile]: I changed it.

**Review by nadavspinco — APPROVED** [review]

**Review by shiraOs — APPROVED** [review]

**Review by Leeadar — APPROVED** [review]

**Review by Yarboa — COMMENTED** [review]
Please refer this comment.
I assume it is related more to your PR
https://github.com/beyond-io/picATrip/pull/14#pullrequestreview-608607287

**Review by Yarboa — APPROVED** [review]

**Pull request merged**